packages/design-system/src/components/popup/utils/getOffset.js

```diff
@@ -134,12 +129,7 @@ export function getOffset({
   const { x: spacingH = 0, y: spacingV = 0 } = spacing || {};
 
   // Horizontal
-  const offsetX = () => {
-    if (resetXOffset && popupRect?.left <= 0) {
-      return isRTL ? width : 0;
-    }
```

Comment:
Previously, this was to handle the color picker in the floating menu when in RTL. We couldn't directly update the offsets since `Tooltip` can sometimes come back with `popupRect?.left` being in the negative. This happen in the layers panel due to the nested absolute positioning. Now, Tooltip handles its own updated offset logic to deal account for this possibility.